test/test_helper.rb

```diff
@@ -38,6 +38,12 @@ def mkdir(dir)
     FileUtils.mkdir path
   end
 
+  def mkdir_p(dir)
```

Comment:
I feel like having both `mkdir` and `mkdir_p` is a bit unnecessary, but it's better than two calls.

Reply:
TIL: `FileUtils.mkdir_p`.

Could we just update the existing `mkdir` to call `FileUtils.mkdir_p`? It shouldn't break the existing behavior.

Reply:
We could, yeah. I don't think it'd be surprising either, so I think I'll do that.